src/components/Palettes/Palettes.js

```diff
@@ -20,11 +20,12 @@ export class Palettes extends Component {
 		const colors = this.props.palette.reduce((acc, colorObject, index) => {
 
 			acc[`color_${index+1}`] = colorObject.color.slice(1)
+			// ^^ making key with color 1,2,3,4,5 and assigning to numbers without #
 			return acc
 		}, {})
 		
 		const newPalette = {...colors, name: this.state.name, project_id: this.props.project.id}
-		this.props.addPalette(newPalette)
+		// this.props.addPalette(newPalette)
```

Comment:
Is this being called somewhere else, or does it need to be completely taken out?